landing-pages/site/content/en/blog/airflow-survey.md

```diff
@@ -0,0 +1,333 @@
+---
+title: "Airflow Survey 2019"
+linkTitle: "Airflow Survey 2019"
+author: "Tomek Urbaszek"
+twitter: "Nuclearriot"
+github: "nuclearpinguin"
+linkedin: "tomaszurbaszek"
+description: "Receiving and adjusting to our users’ feedback is a must. Let’s see who Airflow users are, how they play with it and what they miss."
+tags: ["community", "survey", "users"]
+date: "2019-12-11"
+---
+# Airflow Survey 2019
```

Comment:
Can we call this Apache Airflow in the heading just once?

Reply:
Done